cmd/run.go

```diff
@@ -103,6 +103,50 @@ var runCmd = &cobra.Command{
 	},
 }
 
+var retryCmd = &cobra.Command{
+	Use:   "retry",
+	Short: "Retry a failed setup from jobID [job-id]",
+	Args:  cobra.ExactArgs(1),
+	RunE: func(cmd *cobra.Command, args []string) error {
+		r := buildRunner()
+		defer r.cleanRunner()
+
+		jobID := args[0]
+
+		// check db for existing job
+		job := r.db.GetJob(jobID)
+		if job == nil {
+			return fmt.Errorf("could not find job with id %s", jobID)
+		}
+
+		r.job = job
+		// pull worker ID out
+
+		attachedInstanceIDs, err := job.GetInstanceIds()
+		if err != nil {
+			return err
+		}
+
+		var workerID string
+
+		// this should be way better. ideally the spun up instances themselves
+		// should have
+		for _, i := range attachedInstanceIDs {
+			instance := r.db.GetInstanceByCedanaID(i.InstanceID)
+			if instance.Tag == "worker" {
```

Comment:
eventually we should have multiple instances, but not right now. Going to leave it as it is, because we might have to rewrite a bunch of stuff when dealing with clusters